# HG changeset patch
# User Mike Miller <mtmiller@octave.org>
# Date 1548182986 28800
#      Tue Jan 22 10:49:46 2019 -0800
# Branch stable
# Node ID 988118822a9297fb03bc25f2d6e310625e36f6bf
# Parent  ee21e2bf3e5048aefcc70aabfc84d70ab974b715
shg: fix syntax error introduced in cset b785394f10d0

* shg.m: Delete extraneous parenthesis left over from copy/paste.

diff --git a/scripts/plot/util/shg.m b/scripts/plot/util/shg.m
--- a/scripts/plot/util/shg.m
+++ b/scripts/plot/util/shg.m
@@ -34,12 +34,12 @@ function shg ()
 
   if (nargin != 0)
     warning ("shg: ignoring extra arguments");
   endif
 
   hf = get (0, "currentfigure");
   if (! isempty (hf))
     set (hf, "visible", "on");
-    __show_figure__ (hf));
+    __show_figure__ (hf);
   endif
 
 endfunction
